doc/atari.sgml

```diff
@@ -634,7 +634,18 @@ interface module.
 
 <sect>Limitations<p>
 
-<sect1><tt/atarixl/<#if output="info|latex2e"> limitations</#if><label id="limitations"<p>
+<sect1><tt/Realtime clock/<label id="realtimeclock"<p>
+
+Access to the real time clock is supported only when running on SpartaDOS-X.
+There needs to be an RTC driver installed. This is normally the case
```

Comment:
I deliberately stayed clear from the abbreviation 'RTC' in the docs for the Apple II and the C64. Maybe you want to adhere to that here too.